### PR TITLE
Feature three

### PR DESCRIPTION
**Pull request by Maxwell011**
### Hello in this PR I did the following 

- Import Redux use dispatch.
- Use use selector to consume the state.
- Followed all requirement.

Kindly Review.

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -1,19 +1,56 @@
...
+    title: '',
+    author: '',
+    id: `${Date.now()}`,
+  });
+  const dispatch = useDispatch();
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    dispatch(addBook(book));
+  };
```
Comment (m05e5) [inline src/components/AddBook.js:17]: - [ ] Hi. Currently, when trying to add a new book, the book state does not change and it adds a book without a name or title and the id remains the same please kindly fix it
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Maxwell011 , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 1 above)
Comment (Maxwell011) [reply src/components/AddBook.js:17]: Resolved

Comment (rloterh) [reply src/components/AddBook.js:17]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Maxwell,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- State management using Redux properly implemented

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**